docs/docs/fundamentals/about.md

```diff
@@ -33,40 +33,18 @@ Reanimated aims to provide ways of offloading animation and event handling logic
 
-### Library overview
-
-Version 1 of Reanimated has been designed to match React Native's Animated API while providing a more complete set of primitives for defining interactions.
-In version 2 we decided to change the approach in order to address some limitations of version 1 that come from the declarative API design.
-Below we present an overview of things that are new in Reanimated 2 and different from Reanimated 1.
-This is a tl;dr of the remaining parts of the documentation.
-We recommend that you check the full articles to learn the details about each of the listed aspects:
-
-1. interactions and animations are no longer written using an unintuitive declarative API, instead they can be written in pure JS, in the form of so-called "worklets".
-   Worklets are pieces of JS code that we extract from the main React Native code and run in a separate JS context on the main thread.
-   Because of that, worklets have some limitations as to what part of the JS context they can access (we don't want to load the entire JS bundle into the context which runs on the UI thread).
-2. It is still possible to define and pass around "Animated Values", however thanks to the new API, we expect that you'll create much fewer of those for a single animation.
-   Also, now, they are actually called "Shared Values" and can carry not only primitive types but also arrays, objects and functions.
-3. Shared Values are no longer directly connected to view props.
-   Instead, we expose a `useAnimatedStyle` hook that returns a style object which can be passed as a View's style param.
-   The `useAnimatedStyle` hook takes a worklet that, when executed, should return styles that will be applied to the connected View.
-   The style worklet will update whenever shared values used by that worklet change (we detect dependencies on shared values automatically).
-4. Animations can be started in two ways: by triggering animated change on a shared value, or by returning animated value from `useAnimatedStyle` hook.
-5. With reanimated, we can hook worklets to serve as event handlers.
-   Most common case for an event worklet is to modify some shared values.
-   As a result, changes made to those values will be reflected in the animated style worklet being triggered, which in turn will result in some view properties being updated.
-   For convenience, Reanimated provides an event hook that is tailored to work together with Gesture Handler library and allows you to define a separate worklet for handling different handler states (e.g., onStart, onActive, etc.)
+### Overview of Reanimated 3.x
 
-### Known problems and limitations
+The developments of Reanimated 3 were focused on improving the stability and performance. This version uses the Reanimated v2 API you know and love continuously embracing worklets and shared value architecture.
```

Comment:
Actually, initially the only reason to release v3 was that we didn't want to implement Fabric support for v1 API and we wanted to get rid of the v1 code. After some time, v2 and v3 branches diverged a bit. When we rewrote layout animations and shared values, we didn't want to waste our time on backporting the changes. So effectively v3 comes with stability and performance improvements as well as Fabric support.

Reply:
The effort you guys put into developments of Reanimated 3 resulted in stability and performance improvements. It sounds weird to emphasise "that was just a side effect".

Do you want me to rephrase the sentences to better represent 'the truth' or leave it as it is?

Reply:
Nah, it's okay as it is. Just wanted to tell you the whole story.